stock_move_assign_picking_hook/models/stock_move.py

```diff
@@ -19,6 +19,11 @@ def _domain_search_picking_for_assignation(self):
         return self._domain_search_picking_for_assignation_default()
 
     def _domain_search_picking_for_assignation_default(self):
+        # The hook has been backported into Odoo v14
+        # https://github.com/odoo/odoo/pull/111668
+        # Making this module obsolete.
```

Comment:
This should be mentioned in the ROADMAP I guess to not migrate this module to 15.0+.

Reply:
It has already been mentioned on the migration todo https://github.com/OCA/stock-logistics-workflow/issues/913
as  `stock_move_assign_picking_hook` merged into stock